calypso/proto.go

```diff
@@ -46,13 +46,25 @@ type Write struct {
 	LTSID []byte
 }
 
+// WriteReply is returned upon successfully spawning a Write instance.
+type WriteReply struct {
+	*byzcoin.AddTxResponse
+	byzcoin.InstanceID
+}
+
 // Read is the data stored in a read instance. It has a pointer to the write
 // instance and the public key used to create the read instance.
 type Read struct {
 	Write byzcoin.InstanceID
 	Xc    kyber.Point
 }
 
+// ReadReply is is returned upon successfully spawning a Read instance.
+type ReadReply struct {
+	*byzcoin.AddTxResponse
+	byzcoin.InstanceID
+}
```

Comment:
the structures in `proto.go` are used to generate protobuf files and java implementation. but `WriteReply` and `ReadReply` are not sent over the wire, so they don't need to be here. just keeping these in `api.go` is enough

Reply:
thanks, moving them